### ISSUE TITLE
Request to limit Bluedog Design Bureau backwards compatibility to 1.7.3

### ISSUE THREAD

**Issue by zorg2044** [KSP-CKAN/NetKAN#7451]
Sorry me again. Turns out that a key dependency the newest version B9PartSwitch (BDB uses features specific to that version) is not going to get backwards compatibility below 1.7.3. 

As such would be great if you could kindly set BDB backwards compatibility minimum version to KSP 1.7.3

Thanks a lot.

https://github.com/KSP-CKAN/CKAN-meta/tree/master/BluedogDB

**Comment by DasSkelett** [comment]
If you change the remote avc .version file, the bot should pick it up automatically:
https://github.com/CobaltWolf/Bluedog-Design-Bureau/blob/master/Gamedata/Bluedog_DB/Bluedog_DB.version

**Comment by zorg2044** [comment]
Great. thanks. I misunderstood and thought the change you made last time was manual.

**Comment by DasSkelett** [comment]
No, if you look at what [I changed in the PR I linked](https://github.com/KSP-CKAN/NetKAN/pull/7450/files), I added a cryptic looking line in the .netkan file, which is CKAN's way of telling the bot to look at .version file, and use it to set the version compatibility.
See our Spec here: https://github.com/KSP-CKAN/CKAN/blob/master/Spec.md#ckanksp-avcpathavcfilenameversion

Furthermore the bot tries to use the remote file specified with the URL property. It will overwrite the .version file embedded in the zip.

The second link was just what the bot committed after I added the `#/ckan/ksp-avc`.
